### PR TITLE
Adds surface area statistic

### PR DESCRIPTION
**Pull request by OnTheMF**
These commits add display of the model surface area under "Other Statistics".

### CODE REVIEW

**Comment by hroncok** [comment]
Thanks for you contribution.

Could you please squash your commits together and add the author information to it?

File: src/util.c [hunk 1]
```diff
@@ -388,11 +394,28 @@ static float get_volume(stl_file *stl) {
   return volume;
 }
 
+static float get_surface_area(stl_file *stl) {
+  long i;
```
Comment (hroncok) [inline src/util.c]: since number of facets is just int, I's suggest using int as well

File: src/util.c [hunk 2]
```diff
@@ -540,12 +563,17 @@ All facets connected.  No further nearby check necessary.\n");
...
+    printf("Calculate surfacea area...\n");
+  stl_calculate_surface_area(stl);
+
   if(fixall_flag) {
     if(stl->stats.volume < 0.0) {
       if (verbose_flag)
         printf("Reversing all facets because volume is negative...\n");
       stl_reverse_all_facets(stl);
       stl->stats.volume = -stl->stats.volume;
+      stl->stats.surface_area = -stl->stats.surface_area;
```
Comment (hroncok) [inline src/util.c]: If the volume is negative, you set surface area to it's negative value. What is the reason? The surface are is not negative as well.

**Comment by hroncok** [comment]
did you close this intentionally?

**Comment by OnTheMF** [comment]
Yes, my apologies I pulled from the wrong account. I will make another pull
request this afternoon with the new changes.

Thank,
Matt

**Pull request closed without merging**